datacube/drivers/postgis/_core.py

```diff
@@ -266,8 +265,8 @@ def to_pg_role(role):
 
     Why are we even doing this? Can't we use the same names internally and externally?
```

Comment:
Would now be a good time to address this?

Reply:
Nah. Keep as is until postgres is gone, IMO.